packages/jest-message-util/src/index.ts

```diff
@@ -145,6 +147,27 @@ export const formatExecError = (
       typeof error.stack === 'string'
         ? error.stack
         : `thrown: ${prettyFormat(error, {maxDepth: 3})}`;
+    if ('cause' in error) {
+      const prefix = '\n\nCause:\n';
+      if (typeof error.cause === 'string') {
+        cause += `${prefix}${error.cause}`;
+      } else if (typeof error.cause === 'number') {
+        cause += `${prefix}${error.cause.toString()}`;
```

Comment:
do we need the `toString` call?

Reply:
Oh, that's right. We don't. 😀
I'll push an updated version soon!